src/main/java/org/folio/circulation/domain/UserRepository.java

```diff
@@ -139,7 +140,7 @@ public UserRepository(Clients clients) {
     String propertyName) {
 
     return CqlQuery.exactMatch("barcode", barcode)
-      .after(query -> usersStorageClient.getMany(query, 1))
+      .after(query -> usersStorageClient.getMany(query, Limit.limit(1)))
```

Comment:
I'd extract it as a constant or add to the limit class some frequently used limit constants, e.g. Limit.ONE, etc. (similar to BigDecimal.ZERO/ONE etc.)

Reply:
Sure, I can make that change.